README.rst

```diff
@@ -177,6 +177,27 @@ currently the compiler on these Macintosh Personal Computers do not support open
 The compiler is also gives extra warnings, and remove `-Werror` from the Makefile.local
 is recommended.
 
+Docker
+------
+
+There is a basic docker configuration file to set up a container
+for FastPM. 
+
+To build it, run:
+
+.. code::
+
+    docker build -t fastpm .
+
+To start the docker container in interactive mode,
+with port 8888 exposed and with
+``/my/file/directory`` linked, run
```

Comment:
Directory on the host. E.g., if I have some data directory as `/home/miles/Documents/data`, then I could link this into the docker container with `-v /home/miles/Documents/data:/workspace`. 
Then when I `cd /workspace` inside the docker container, I see all the files in `/home/miles/Documents/data`. If nothing is linked from the host, then any file I create inside the container will be deleted after removing the container.